src/crud.py

```
@@ -1611,6 +1613,78 @@ async def create_document(
+        .where(models.Document.peer_name == peer_name)
+        .where(models.Document.collection_name == collection_name)
+        .where(models.Document.public_id == document_id)
+    )
+    result = await db.execute(stmt)
+    document = result.scalar_one_or_none()
+    if document is None:
+        raise ResourceNotFoundException("Document not found")
+    return document
+
+
+async def update_document(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)